src/datahike/index/hitchhiker_tree.cljc

```diff
@@ -1,5 +1,5 @@
 (ns datahike.index.hitchhiker-tree
-  (:require [hitchhiker.tree.utils.async :as async]
+  (:require [hitchhiker.tree.utils.clojure.async :as async]
```

Comment:
Does that have anything to do with 'update deps.edn'? Remove that change?

Reply:
The latest hitchhiker-tree requires this change. This was an attempt to avoid future conflicts. Thinking we just close this one and include the migration in the async work to reduce the number of branches we have to merge.

Reply:
I prefer to keep the changes small and frequent. Feel free to close this one but @elMor3no will change the CI to deps.edn together with me and the deps.edn will then be merged.